README.md

```diff
@@ -34,7 +37,11 @@ gulp.src('./dist/**')
 	.pipe( appPublisher.getStream() );
 ```
 
-Alternately, to publish a library (e.g. jQuery, Angular, etc.) to the CDN:
+**Note**: Make sure the version number for your application follows the guideline specified in [Semantic Versioning](http://semver.org).
```

Comment:
This restriction only applies to release version - i.e. it only applies to the `version` parameter, not to the `devTag` parameter.  I don't think this is sufficiently clear.